README.md

```diff
@@ -59,6 +59,30 @@ packages via the command line.
   * [Fedora package] stable release: `# dnf install freeorion`
   * [ArchLinux package] stable release
 
+## Directories
+* install directory  
+The location of this README.md file, called `<install_dir>` below.  
+* config - game settings  
+    * linux  
+`$XDG_CONFIG_HOME/freeorion` which defaults to `~/.config/freeorion`  
+    * OSX  
+`Username/Library/Application Support/FreeOrion/`  
+    * Windows  
+`C:\Users\Username\AppData\Roaming\FreeOrion`  
+* data - local user data, saved games, log files  
+    * linux  
+`$XDG_DATA_HOME/freeorion` which defaults to `~/.config/freeorion`  
+    * OSX  
+`Username/Library/Application Support/FreeOrion/`  
+    * Windows  
+`C:\Users\Username\AppData\Roaming\FreeOrion`  
+* stringtables - translation indices for various languages  
+    `<install_dir>/default/stringtables/`  
```

Comment:
I added the resource directory with a comment that it can be relocated in `config.xml` or `persistent_config.xml`